pkg/internal/infrastructure/terraform.go

```diff
@@ -116,22 +123,9 @@ func ComputeTerraformerChartValues(infra *extensionsv1alpha1.Infrastructure, cli
 		outputKeys["availabilitySetID"] = TerraformerOutputKeyAvailabilitySetID
 		outputKeys["availabilitySetName"] = TerraformerOutputKeyAvailabilitySetName
 
-		cloudProfileConfig, err := helper.CloudProfileConfigFromCluster(cluster)
-		if err != nil {
-			return nil, err
+		if err := fillDomainCounts(azure, cluster, infra); err != nil {
```

Comment:
Style-wise, I would not pass the `azure` and let the function rather return the two values and write it into `azure` here. Then there is only "one place"/function where the Helm chart values are computed. Now you have to look into multiple functions to find where a certain value is coming from. No strong opinion/request to change, though.

Reply:
Okay, agree 👍